src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonUtilities.Defaults.cs

```diff
@@ -16,6 +17,19 @@ public static partial class AIJsonUtilities
     /// <summary>Gets the <see cref="JsonSerializerOptions"/> singleton used as the default in JSON serialization operations.</summary>
     public static JsonSerializerOptions DefaultOptions { get; } = CreateDefaultOptions();
 
+    /// <summary>
+    /// Gets a strongly typed <see cref="JsonTypeInfo{T}"/> instance for the specified <typeparamref name="T"/>.
+    /// </summary>
+    /// <typeparam name="T">The type for which to resolve the JSON contract.</typeparam>
+    /// <param name="options">The options from which to resolve the JSON metadata.</param>
+    /// <returns>A <see cref="JsonTypeInfo{T}"/> instance for the specified type.</returns>
+    public static JsonTypeInfo<T> GetTypeInfo<T>(this JsonSerializerOptions options)
```

Comment:
On the fence on whether this should be an extension method for JSO. It's a fairly useful helper when building AOT compatible libraries.

Reply:
I don't think M.E.AI should be in the business of supplying non-MEAI-related extension methods onto non-MEAI types.

Reply:
Does keeping it as a static seem fair? Or should we just remove it/replicate it in the leaf clients?

Reply:
Removed.